assets/styles/parallax.css

```diff
@@ -0,0 +1,40 @@
+[class*="parallax-"] {
```

Comment:
I love the effects you're able to create with this CSS! Super cool and really advanced!